.github/workflows/storybook.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and Deploy
+on:
+  push:
+    paths: ['stories/**', 'src/components/**'] # Trigger the action only when files change in the folders defined here
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.1
```

Comment:
`@v2.3.1` is OK, but maybe `@2` is just fine? (See other GH workflows.)

The same goes for other actions in all workflows — unless we really want to use a specific version. Either is fine but it should be the same way everywhere.

Reply:
For checkout us `@v2` just fine. But for `JamesIves/github-pages-deploy-action` the `@4` does not work. So I must stay with `@4.1.4`.

Reply:
I'm a little bit confused with versions you are mentioning, but I believe it's a typo and I get the point 🙂.